### PR TITLE
enh(twig): update keywords list

### PR DESCRIPTION
**Pull request by MrYamous**
### Changes
Update keywords list for Twig according to [documentation](https://twig.symfony.com/doc/3.x/)

### Checklist
- [x] Added markup tests, or they don't apply here because it's about updating keywords list
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Grammars:
 
+- enh(twig) update keywords list
```
Comment (joshgoebel) [inline CHANGES.md:5]: Please credit yourself here and on line ~30.... (as with other examples)

Comment (MrYamous) [reply CHANGES.md]: Thanks for your review, it should be better with the last commit

**Comment by joshgoebel** [comment]
I've cleaned up the Twig support a lot in my follow-up commit, any chance you could take a look and provide your thoughts?

**Comment by joshgoebel** [comment]
Ping.

**Comment by MrYamous** [comment]
I also noticed Symfony's own twig functions and filters are not in the keywords list for the moment (visible in [Symfony's doc](https://symfony.com/doc/current/reference/twig_reference.html) and [Twig's doc](https://twig.symfony.com/doc/3.x/))
Do you think it's too specific or can I add them? Maybe with an other PR

**Pull request merged**